tsconfig.json

```diff
@@ -11,13 +11,8 @@
     "moduleResolution": "node",
     "importHelpers": true,
     "target": "es2015",
-    "typeRoots": [
-      "node_modules/@types"
-    ],
-    "lib": [
-      "es2018",
-      "dom"
-    ]
+    "typeRoots": ["node_modules/@types"],
```

Comment:
dont change existing formatting.
I can see your changed this at lots of places

Reply:
Done